### PR TITLE
Update repository & CONTRIBUTING links

### PR DESCRIPTION
**Pull request by mattnorris**
# Pull Request 

## Description

Updated GitHub repository links and CONTRIBUTING.md to work on npmjs.com.

Fixes https://jira-eng-gpk2.cisco.com/jira/browse/SSDK-1934

## Type of Change

Please delete options that are not relevant.

- [x] This change requires a documentation update

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: packages/node_modules/@ciscospark/internal-plugin-board/package.json [hunk 1]
```diff
@@ -6,7 +6,7 @@
   "author": "Greg Hewett <ghewett@cisco.com>",
   "main": "dist/index.js",
   "devMain": "src/index.js",
-  "repository": "https://github.com/webex/spark-js-sdk/tree/master/packages/plugin-board",
+  "repository": "https://github.com/webex/spark-js-sdk/tree/master/packages/node_modules/@ciscospark/plugin-board",
```
Comment (bbender) [inline packages/node_modules/@ciscospark/internal-plugin-board/package.json]: `https://github.com/webex/spark-js-sdk/tree/master/packages/node_modules/@ciscospark/internal-plugin-board`

File: packages/node_modules/@ciscospark/internal-plugin-calendar/package.json [hunk 2]
```diff
@@ -5,7 +5,7 @@
   "author": "Uday Srinath <usrinath@cisco.com>",
   "main": "dist/index.js",
   "devMain": "src/index.js",
-  "repository": "https://github.com/webex/spark-js-sdk/tree/master/packages/plugin-calendar",
+  "repository": "https://github.com/webex/spark-js-sdk/tree/master/packages/node_modules/@ciscospark/plugin-calendar",
```
Comment (bbender) [inline packages/node_modules/@ciscospark/internal-plugin-calendar/package.json]: `https://github.com/webex/spark-js-sdk/tree/master/packages/node_modules/@ciscospark/internal-plugin-calendar`

File: packages/node_modules/@ciscospark/internal-plugin-conversation/package.json [hunk 3]
```diff
@@ -6,7 +6,7 @@
   "author": "Ian W. Remmel <iremmel@cisco.com>",
   "main": "dist/index.js",
   "devMain": "src/index.js",
-  "repository": "https://github.com/webex/spark-js-sdk/tree/master/packages/plugin-conversation",
+  "repository": "https://github.com/webex/spark-js-sdk/tree/master/packages/node_modules/@ciscospark/plugin-conversation",
```
Comment (bbender) [inline packages/node_modules/@ciscospark/internal-plugin-conversation/package.json]: `https://github.com/webex/spark-js-sdk/tree/master/packages/node_modules/@ciscospark/internal-plugin-conversation`

File: packages/node_modules/@ciscospark/internal-plugin-encryption/package.json [hunk 4]
```diff
@@ -6,7 +6,7 @@
   "author": "Ian W. Remmel <iremmel@cisco.com>",
   "main": "dist/index.js",
   "devMain": "src/index.js",
-  "repository": "https://github.com/webex/spark-js-sdk/tree/master/packages/plugin-encryption",
+  "repository": "https://github.com/webex/spark-js-sdk/tree/master/packages/node_modules/@ciscospark/plugin-encryption",
```
Comment (bbender) [inline packages/node_modules/@ciscospark/internal-plugin-encryption/package.json]: `https://github.com/webex/spark-js-sdk/tree/master/packages/node_modules/@ciscospark/internal-plugin-encryption`

File: packages/node_modules/samples/package.json [hunk 5]
```diff
@@ -4,7 +4,7 @@
   "description": "Sample apps for the CiscoSpark JavaScript SDK",
   "license": "MIT",
   "author": "Ian W. Remmel <iremmel@cisco.com>",
-  "repository": "https://github.com/webex/spark-js-sdk/tree/master/packages/samples",
+  "repository": "https://github.com/webex/spark-js-sdk/tree/master/packages/node_modules/@ciscospark/samples",
```
Comment (bbender) [inline packages/node_modules/samples/package.json]: `https://github.com/webex/spark-js-sdk/tree/master/packages/node_modules/samples`

File: packages/node_modules/ciscospark/package.json [hunk 6]
```diff
@@ -6,7 +6,7 @@
   "author": "Ian W. Remmel <iremmel@cisco.com>",
   "main": "dist/index.js",
   "devMain": "src/index.js",
-  "repository": "https://github.com/webex/spark-js-sdk/tree/master/packages/ciscospark",
+  "repository": "https://github.com/webex/spark-js-sdk/tree/master/packages/node_modules/@ciscospark/ciscospark",
```
Comment (bbender) [inline packages/node_modules/ciscospark/package.json]: `https://github.com/webex/spark-js-sdk/tree/master/packages/node_modules/ciscospark`

**Comment by mattnorris** [comment]
Great catches, @bbender - thank you!

**Pull request merged**